examples/neural_compressor/language-modeling/run_clm.py

```diff
@@ -223,7 +219,6 @@ class OptimizationArguments:
         metadata={"help": "Scheme for weight only quantization. Choose from 'sym' and 'asym'."},
     )
     quantization_methodology: str = field(
-        choices=["rtn", "gptq"],

```

Comment:
why removed ?

Reply:
I got error
```
  File "/home/changwa1/optimum-intel-chang/examples/neural_compressor/language-modeling/run_clm.py", line 221, in OptimizationArguments
    quantization_methodology: str = field(
TypeError: field() got an unexpected keyword argument 'choices'
```

Reply:
oh right there's no choices in dataclass fields, I think it was copied from an `argparser.add_argument`